src/app/notification/notification.component.ts

```diff
@@ -35,6 +44,21 @@ export class NotificationComponent implements OnDestroy {
     }
   }
 
+  public answerPrompt(index: number): void {
+    this.http
+      .post(
+        this.configService.getApiURL('plugin/action_command_prompt'),
+        { command: 'select', choice: index },
+        this.configService.getHTTPHeaders()
+      )
+      .pipe(
+        catchError(error =>
+          this.notificationService.setError($localize`:@@error-answer-prompt:Can't answer prompt!`, error.message),
+        ),
+      )
+      .subscribe();
+  }
```

Comment:
this should be in `system-service` (or another service if there's one that's better fitting) since it won't be compatible with Klipper / other interfaces. 

From now on all communication with OctoPrint should be behind an abstract service with a generic interface to enable easy integration of other printer software.